lib/streamlit/runtime/state/safe_session_state.py

```diff
@@ -74,6 +74,13 @@ def on_script_finished(self, widget_ids_this_run: Set[str]) -> None:
 
             self._state.on_script_finished(widget_ids_this_run)
 
+    def maybe_check_serializable(self) -> None:
```

Comment:
super nit: I think the name `maybe_enforce_serializable` (here and in `session_state.py`) is a bit nicer now that we use the word enforce in the config option name